src/router.js

```diff
@@ -45,9 +54,13 @@ export default Router
 
 // check if feature is allowed and redirect if not
 async function featureAvailable(apollo, networkId, to) {
-  const feature = `feature_${to.meta.feature.toLowerCase()}`
-  const { data } = await apollo.query({
-    query: NetworkCapability(networkId)
-  })
-  return NetworkCapabilityResult(feature)(data)
+  if (to.meta.feature === undefined) {
+    return
+  } else {
+    const feature = `feature_${to.meta.feature.toLowerCase()}`
+    const { data } = await apollo.query({
```

Comment:
you should get this from the networks object and not do another query

Reply:
I didn't add this query. It was already [there ](https://github.com/luniehq/lunie/blob/5648abce07c8746d6856ea6d21ac9e7b544e3b51/src/router.js#L49)

Reply:
Also, there is not network object right now in this file. Are you suggesting using the two getters (networks and network) to get it?

Reply:
> Are you suggesting using the two getters (networks and network) to get it?

Yes, like everywhere else

Reply:
Done